RecoParticleFlow/PFProducer/interface/PFCandConnector.h

```diff
@@ -61,21 +61,20 @@ class PFCandConnector {
 
 
        void setParameters(bool bCorrect, bool bCalibPrimary, double dptRel_PrimaryTrack, double dptRel_MergedTrack, double ptErrorSecondary, const std::vector<double>& nuclCalibFactors);
-       void setDebug( bool debug ) {debug_ = debug;}
 
        
 
-       std::unique_ptr<reco::PFCandidateCollection> connect(std::unique_ptr<reco::PFCandidateCollection>& pfCand);
+       reco::PFCandidateCollection connect(reco::PFCandidateCollection& pfCand) const;
 
        
  
     private :
 
        /// Analyse nuclear interactions where a primary or merged track is present
-       void analyseNuclearWPrim(std::unique_ptr<reco::PFCandidateCollection>&, unsigned int);
+       void analyseNuclearWPrim(reco::PFCandidateCollection&, std::vector<bool> &, unsigned int) const;
 
```

Comment:
I updated with force-push because my mistake that caused the differences was not interesting enough for it's own commit: I forgot to pass the mask vector by reference (also in the other function)...